modules/telco-upgrade-acknowledging-the-upgrade.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-acknowledging-upgrade-for-eus-eus-y-stream_{context}"]
+= Acknowledging the upgrade for EUS-EUS and y-stream upgrades
+
+When you upgrade to all versions from 4.11 and later, you need to manually acknowledge that the upgrade can continue.
+
+.Procedure
+. Run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-<upgrade_version_from>-kube-<kube_api_version>-api-removals-in-<upgrade_version_to>":"true"}}' --type=merge
```

Comment:
Before they do this admin-ack, they're supposed to actually verify that there are no Kubernetes APIs in use that are removed in the version they're updating to. Probably should make mention of this before just having them do this ack.

https://docs.openshift.com/container-platform/4.16/updating/preparing_for_updates/updating-cluster-prepare.html#kube-api-removals_updating-cluster-prepare

And this only applies when updating to certain versions. They will not have to do it for 4.17 (https://docs.openshift.com/container-platform/4.17/updating/preparing_for_updates/updating-cluster-prepare.html#kube-api-removals_updating-cluster-prepare), but would for updating to 4.16.

Reply:
Thank you! this is a great catch